shuttle/index.php

```diff
@@ -0,0 +1,45 @@
+<?php
+use ThauEx\SimpleHtmlDom\SHD;
+
+SHD::$fileCacheDir = "cache";
+$html = SHD::fileGetHtml('http://wzw.tum.de/index.php?id=416');
+if (empty($html)) {
+	// something wrong with fileGetHTML
+	// redirect to overview page than
+	header('Location: http://wzw.tum.de/index.php?id=416');
+	die();
+}
+
+$links = array();
+
+foreach($html->find('div[id=c2489]') as $element) {
+	// inside the div#c2489 there should be one to three (or more) links
+	foreach($element->find('a') as $element2) {
+       		$links[] = $element2->href;
```

Comment:
Why is this indented like 3 times as much as it would be? Please use four spaces, as per all our files are formatted that manner.